edit-post/assets/stylesheets/_mixins.scss

```diff
@@ -209,6 +209,12 @@ $float-margin: calc( 50% - #{ $visual-editor-max-width-padding / 2 } );
 			left: $admin-sidebar-width;
 		}
 	}
+	
+	.edit-post-layout #{$selector} { /* Without auto-fold or folded */
```

Comment:
This is supposed to be a generic mixin, we don't want to target specific divs here? Does it work if we just remove this selector here? Keep only: `#{$selector} {`

Reply:
It works for `edit-post-header`, but not for `components-notice-list`, because https://github.com/WordPress/gutenberg/blob/master/edit-post/components/layout/style.scss#L6 will override it with `left: 0;`. Do you have any suggestions?

Reply:
Personally I don't think the notices list should be sticky, as it overlaps both the scrollbar and the block sticky toolbar. Ideally it'd just be a static panel located below the header toolbar, but I'd imagine based on the sticky behavior of the block toolbar, overlapping might still occur.